sdk/nodejs/image.ts

```diff
@@ -101,7 +101,7 @@ export class Image extends pulumi.ComponentResource {
     public digest: pulumi.Output<string | undefined>;
 
     constructor(name: string, args: ImageArgs, opts?: pulumi.ComponentResourceOptions) {
-        super("docker:image:Image", name, argsWithoutRegistry(args), opts);
+        super("docker:image:Image", name, {}, opts);
```

Comment:
We've made a change in pulumi/pulumi a while back that **component** resources entirely ignore hte 'props' arg.  So this munging of args in this layer is no longer necessary.